client/app/core/catalogs-state.service.js

```diff
@@ -97,7 +97,7 @@ export function CatalogsStateFactory(CollectionsApi, EventNotifications, sprintf
     }
 
     function createFailure(response) {
-      EventNotifications.error(sprintf(__("There was an error creating catalog %s."), catalogObj.name));
+      EventNotifications.error(__(response.data.error.message));
```

Comment:
oooooo down with that, yeah server kicks back a message we can use, don't think server cares much for translation... or does it? 

@jntullo you seen server changing response messages based on language?